src/libraries/System.Linq/src/System/Linq/Zip.cs

```diff
@@ -42,6 +42,36 @@ public static IEnumerable<(TFirst First, TSecond Second)> Zip<TFirst, TSecond>(t
             return ZipIterator(first, second);
         }
 
+        /// <summary>
+        /// Produces a sequence of tuples with elements from the three specified sequences.
+        /// </summary>
+        /// <typeparam name="TFirst">The type of the elements of the first input sequence.</typeparam>
+        /// <typeparam name="TSecond">The type of the elements of the second input sequence.</typeparam>
+        /// <typeparam name="TThird">The type of the elements of the third input sequence.</typeparam>
+        /// <param name="first">The first sequence to merge.</param>
+        /// <param name="second">The second sequence to merge.</param>
+        /// <param name="third">The third sequence to merge.</param>
+        /// <returns>A sequence of tuples with elements taken from the first, second and third sequences, in that order.</returns>
```

Comment:
Nit: long live the Oxford comma :)
```suggestion
        /// <returns>A sequence of tuples with elements taken from the first, second, and third sequences, in that order.</returns>
```